@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/CfgCloudlets.hpp

```diff
@@ -195,179 +47,24 @@ class CfgCloudlets
-        particleFSNtieth = 16;
-        particleFSIndex = 12;
-        particleFSFrameCount = 40;
-        particleFSLoop = 1;
-        particleShape = "\A3\data_f\ParticleEffects\Universal\Universal";
-        particleType = "Billboard";
-        position[] = {0,0,0};
-        positionVar[] = {0.5,0.5,0.5};
-        randomDirectionIntensity = 0.2;
-        randomDirectionIntensityVar = 0;
-        randomDirectionPeriod = 0.1;
-        randomDirectionPeriodVar = 0;
-        rubbing = 0.2;
-        size[] = {0.6,1.2,1.5,1.75,1.85,2};
-        sizeCoef = 1;
-        sizeVar = 0.3;
-        timerPeriod = 0.5;
-        volume = 0.1;
-        weight = 0.1275;
-    };
-};
-
-
-class fza_trail_smoke_small_effect
-{
-    class Trail_Smoke_Small
-    {
-        simulation = "particles";
-        type = "fza_trailsmoke_small";
-        position[] = {0,0,0};
-        intensity = 1;
-        interval = 0.1;
-        lifeTime = 10;
-    };
-};
-
-class fza_airburst_effect
-{
-    class Explosion
-    {
-        simulation = "particles";
-        type = "GrenadeExp";
-        position[] = { 0, 0, 0 };
-        intensity = 0.5;
-        interval = 1;
-        lifeTime = 0.5;
-    };
-    class Explosion_Light
-    {
-        simulation = "light";
-        type = "GrenadeExploLight";
-        position[] = { 0, 0, 0 };
-        intensity = 0.01;
-        interval = 1;
-        lifeTime = 0.2;
-    };
-    class Burst_Smoke
-    {
-        simulation = "particles";
-        type = "fza_burst_smoke";
-        position[] = {0,0,0};
-        intensity = 1;
-        interval = 1;
-        lifeTime = 1;
-    };
-    class Trail_Head
-    {
-        simulation = "particles";
-        type = "fza_trailhead";
-        position[] = {0,0,0};
-        intensity = 1;
-        interval = 1;
-        lifeTime = 0.1;
-    };
-};
-
-class fza_smoke_effect
-{
-    class Smoke
-    {
-        simulation = "particles";
-        type = "fza_smoke";
-        position[] = {0,0,10};
-        lifeTime = 120;
-        qualityLevel = -1;
-    };
-};
-
-
-class fza_ah64_rocketsmoke
-{
-    class Light1
-    {
-        simulation = "light";
-        type = "RocketLight";
-        position[] = {0,0,0};
-        intensity = 0.01;
-        interval = 1;
-        lifeTime = 1;
-    };
-    class MissileEffects1
-    {
-        simulation = "particles";
-        type = "Missile1";
-        position[] = {0,0,0};
-        intensity = 1;
-        interval = 1;
-        lifeTime = 1;
-    };
-};
-class fza_ah64_hellfiresmoke
-{
-    class Light1
+    class fzaHydraTrailSmoke: Missile4
     {
-        simulation = "light";
-        type = "RocketLight";
-          position[] = {0,0,0};
-          intensity = 0.01;
-          interval = 1;
-          lifeTime = 1;
-    };
-    class Missile2
-    {
-        simulation = "particles";
-        type = "Missile2";
-        position[] = {0,0,0};
-        intensity = 1;
-        interval = 1;
-        lifeTime = 1;
-    };
-};
-
-class fza_ah64_deploysmoke
```

Comment:
Didn't we just implement deploysmoke - isn't it used for flechettes?

Reply:
oops

Reply:
Corrected

Reply:
You re-added at a different point to where they are so they still show up in the diff - needs moving back above CfgLights

Reply:
corrected